public/css/stylesheet.css

```diff
@@ -229,3 +229,73 @@ Aligns Main Menu under button
   justify-content: space-between;
 }
 /*profile page css end*/
+.form-box {
+    margin-top: 40px;
+    margin-right: 50px;
+    padding-right: 65px;
+   
+}
+.form-top {
+    overflow: hidden;
+    padding: 0 15px 15px 25px;
+    background: #444;
+    background: rgba(0, 0, 0, 0.35);
+    -moz-border-radius: 4px 4px 0 0; -webkit-border-radius: 4px 4px 0 0; border-radius: 4px 4px 0 0;
+    text-align: left;
+}
+.form-top-left {
+    float: left;
+    width: 75%;
+    padding-top: 25px;
+}
+.form-top-left h3 { margin-top: 0; color: #fff; }
+
+.form-top-left p { opacity: 0.8; color: #fff; }
+
+.form-top-right {
+    float: left;
+    width: 25%;
+    padding-top: 0px;
+    font-size: 66px;
+    color: #fff;
+    line-height: 100px;
+    text-align: right;
+    opacity: 0.3;
+}
+
+.form-bottom {
+    padding: 25px 25px 30px 25px;
+    background: #444;
+    background: rgba(0, 0, 0, 0.3);
+    -moz-border-radius: 0 0 4px 4px; -webkit-border-radius: 0 0 4px 4px; border-radius: 0 0 4px 4px;
+    text-align: left;
+}
+
+.form-bottom form textarea {
+    height: 100px;
+}
+
+.form-bottom form button.btn {
+    width: 100%;
+}
+
+.form-bottom form .input-error {
+    border-color: #19b9e7;
+}
+
+.bg {
+    width: 105%;
+    height: 100%;
+    font-family: 'Raleway', sans-serif;
+    margin-left: -15px;
```

Comment:
`margin-left` this should be deleted, the nagtive margin plus the oversized div is causing the overflow outisde of the page